pkg/virt-handler/device-manager/mediated_device.go

```diff
@@ -50,10 +52,58 @@ type MDEV struct {
 }
 
 type MediatedDevicePlugin struct {
-	DevicePluginBase
+	*DevicePluginBase
```

Comment:
Why do you changed this to a pointer?You have missed the socket_
I would also squash this change in the previous commit where you remove the code duplication